src/commands/kitty-review.rs

```diff
@@ -0,0 +1,70 @@
+use serenity::builder::CreateApplicationCommand;
+use serenity::model::prelude::interaction::application_command::ApplicationCommandInteraction;
+use serenity::model::prelude::interaction::InteractionResponseType;
+use serenity::prelude::Context;
+use serenity::utils::Color;
+use tracing::error;
+use i18n::t;
+use reqwest::header;
+
+pub(crate) fn register(
+    command: &mut CreateApplicationCommand,
+) -> &mut CreateApplicationCommand {
+    command
+        .name("kitty-review")
+        .description(t!("commands.kitty-review.description").to_string())
+        .description_localized("cs", t!("commands.kitty-review.description", "cs"))
+        .dm_permission(true)
+}
+
+fn get_message(locale: String) -> &'static str {
+    match locale.as_str() {
+        "cs" => t!("commands.kitty-review.message", "cs"),
+        _ => t!("commands.kitty-review.message"),
+    }
+}
+
+pub(crate) async fn run(ctx: &Context, command: &ApplicationCommandInteraction) {
+    let locale = command.clone().locale;
+
+    let tenor_api_key = "API KEY ZDE";
```

Comment:
API KEY should NEVER be part of your source code, you can either use environment or use the config object (see how discord_token is loaded from Config.toml)